tests/System/client_dms.sh

```diff
@@ -10,8 +10,8 @@ echo "====== dirac-proxy-init -g dteam_user" #this is necesary to upload user fi
 dirac-proxy-init -g dteam_user
 
 dirac_user=$( dirac-proxy-info | awk '/^username / {print $3}' )
-#userdir="/dteam/user/$( echo "$USER" |cut -c 1)/$USER"
-userdir="/dteam/user/$( echo "$dirac_user" |cut -c 1)/$dirac_user"
+#userdir="/dteam/user/$( echo "$USER" | cut -c 1)/$USER"
+userdir="/dteam/user/$( echo "$dirac_user" | cut -c 1)/$dirac_user"
```

Comment:
ideally this should also dynamically find the VO, not hardocde dteam

Reply:
I added this as a comment to https://github.com/DIRACGrid/DIRAC/issues/4631